app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditViewModel.kt

```diff
@@ -26,19 +26,23 @@ class ProfileEditViewModel @Inject constructor(
   /** Specifies whether download access has been enabled by the user. */
   val isAllowedDownloadAccess: LiveData<Boolean> = isAllowedDownloadAccessMutableLiveData
 
+  /** Live data for managing profile. */
   val profile: LiveData<Profile> by lazy {
     Transformations.map(
       profileManagementController.getProfile(profileId).toLiveData(),
       ::processGetProfileResult
     )
   }
 
+  /** Verifies whether the user has admin rights or not. */
   var isAdmin = false
 
+  /** Sets the profile id of all the users. */
   fun setProfileId(id: Int) {
     profileId = ProfileId.newBuilder().setInternalId(id).build()
   }
 
+  /** This fetches the profile of a user asynchronously. */
```

Comment:
Nit, but this doesn't need a KDoc since it's private (though it's also fine to keep if you think it improves readability).

Reply:
Kept it as it tells about the asynchronous behavior of the method.

Reply:
In that case, it should follow the same formatting rules as a regular KDoc (i.e. the first sentence should be a fragment verb phrase that finishes the sentence "This method ...").